src/k8s/pkg/k8sd/features/localpv/chart.go

```diff
@@ -11,11 +11,21 @@ var (
 	chart = helm.InstallableChart{
 		Name:         "ck-storage",
 		Namespace:    "kube-system",
-		ManifestPath: path.Join("charts", "rawfile-csi-0.8.0.tgz"),
+		ManifestPath: path.Join("charts", "rawfile-csi-0.9.0.tgz"),
 	}
 
-	// imageRepo is the image to use for Rawfile LocalPV CSI.
+	// imageRepo is the repository to use for Rawfile LocalPV CSI.
 	imageRepo = "ghcr.io/canonical/rawfile-localpv"
+	// csiImageRepo is the repository to use for the CSI images.
+	csiImageRepo = "ghcr.io/canonical/k8s-snap/sig-storage"
```

Comment:
its ok to not have this as a separate arg, so that individual mages can be changed easily without much trouble